components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -210,9 +204,9 @@ func main() {
 	internalEvalAssistant := avs.NewInternalEvalAssistant(cfg.Avs)
 	externalEvalCreator := provisioning.NewExternalEvalCreator(avsDel, cfg.Avs.Disabled, externalEvalAssistant)
 
-	clientHTTPForIAS := httpClient
+	clientHTTPForIAS := httputil.NewClient(30, cfg.IAS.SkipCertVerification)
 	if cfg.IAS.TLSRenegotiationEnable {
-		clientHTTPForIAS = httputil.NewRenegotiationTLSClient(30, cfg.Director.SkipCertVerification)
+		clientHTTPForIAS = httputil.NewRenegotiationTLSClient(30, cfg.IAS.TLSRenegotiationEnable)
```

Comment:
Second argument of `NewRenegotiationTLSClient` is `skipCertVerification`. `TLSRenegotiationEnable` var is only used to decide if regular HTTP client should be used or with "renegotiation" option.